nodes/ExceptHandler.py

```diff
@@ -1,19 +1,35 @@
 """
 ExceptHandler astroid node
 
+ExceptHandler is a single except clause.
+
 Attributes:
-    - type  (node)  typically a Name node or None for a catch-all "except:"
-    clause
-    - name  (str)   a raw string for the name to hold the exception
-    - body  (list)  a list of nodes
+    - type  (node)
+        - Typically a Name node like ValueError or TypeError.
+    - name  (str)
+        - A raw string for the name to hold the exception.
+    - body  (List[Node])
+        - A list of nodes.
 
-Example:
+Example 1:
     - type  -> None
     - name  -> None
     - body  -> [pass]
+
+Example 2:
+    - type  -> ValueError
+    - name  -> None
+    - body  -> [print("NOOOOOO!!!!!!!!!")]
 """
 
+#Example 1
 try:
     pass
 except:
```

Comment:
Hey Nigel thanks for commenting!
I wasn't sure if I can change the original example and we decided last week that we'll add another example to help beginners understand the purpose of this node. Btw this is not final draft (I committed these changes because I was practicing using git :P)